JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -4,9 +4,32 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-// 과일 저장소 타입
 class FruitStore {
+    private var stockList: [Fruit: Int] = [:]
+    
+    init(stockQuantity: Int = 10) {
+        for fruit in Fruit.allCases {
+            stockList[fruit] = stockQuantity
+        }
+    }
```

Comment:
- 재고 수량의 값을 초기화 단계에서 설정해주셨군요! 인스턴스 생성 시 값을 기입하는 것과 무슨 차이가 있을까요?
- `for문`으로 구현하셨군요. 저라면 `forEach()`로 구현했을 거 같습니다. 
```suggestion
    init(stockQuantity: Int = 10) {
        Fruit.allCases.forEach { stockList[$0] = stockQuantity }
    }
```

Reply:
추후 사용자가 직접 기본값을 변경하게끔 코딩할 수 있다는 점에서 차이가 있습니다.